aws_analysis_tools/cli/convert_ip.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+#
+# © 2016 Krux Digital, Inc.
+#
+
+#
+# Standard libraries
+#
+
+from __future__ import absolute_import
+from pprint import pformat
+
+#
+# Internal libraries
+#
+
+import krux_ec2.cli
+from krux_ec2.filter import Filter
+
+NAME = 'convert_ip'
+
+
+class Application(krux_ec2.cli.Application):
+
+    _IP = 'ip-address'
+    _PRIVATE_IP = 'private-ip-address'
+
+    def __init__(self, name=NAME):
+        # Call to the superclass to bootstrap.
+        super(Application, self).__init__(name=name)
+
+        # Set what AWS arg to filter based on
+        self.filter_arg = Application._PRIVATE_IP if self.args.private else Application._IP
+
+    def add_cli_arguments(self, parser):
+        # Call to the superclass first
```

Comment:
Is this also calling to the superclass to bootstrap? If not, can we put the why in here rather than the what?